orbit/cmd/desktop/desktop.go

```diff
@@ -84,6 +115,42 @@ func main() {
 			}
 		}()
 
+		go func() {
+			tic := time.NewTicker(5 * time.Minute)
+			tr := http.DefaultTransport.(*http.Transport)
+			if os.Getenv("FLEET_DESKTOP_INSECURE") != "" {
+				tr.TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+			}
+			client := &http.Client{
+				Transport: tr,
+			}
+
+			for {
+				<-tic.C
+				resp, err := client.Get(devTestURL.String())
```

Comment:
Let's leave a TODO comment to use the policies endpoint (and the issue number, #5697).